src/core/blocks/core.cpp

```diff
@@ -1006,8 +1006,43 @@ struct Assoc : public VariableBase {
   ExposedInfo _requiredInfo{};
   Type _tableTypes{};
 
+  static inline Parameters params{
+      {"Name",
+       CBCCSTR("The name of the sequence or table to be updated."),
+       {CoreInfo::StringOrAnyVar}},
+      {"Key",
+       CBCCSTR("Table key for the value that is to be updated. Parameter "
+               "applicable if target is table."),
+       {CoreInfo::StringStringVarOrNone}},
+      {"Global",
+       CBCCSTR("If the variable is or should be available to all the chains in "
+               "the same node. The default value (false) makes the variable "
+               "local to the "
+               "chain."),
```

Comment:
It should be on a single line.

Maybe run `ninja format` once more, after putting the whole string on a single line first.

Reply:
Oh, I don't know how I messed that up! Fixed.